### ISSUE TITLE
Update doc for SuppressWithPlainTextCommentFilter

### ISSUE THREAD

**Issue by AbhinavRajputEXE** [checkstyle/checkstyle#11199]
https://checkstyle.org/config_filters.html#SuppressWithPlainTextCommentFilter_Properties
![Screenshot 2022-01-16 025133](https://user-images.githubusercontent.com/80591258/149638117-c58a2178-4182-4e6c-a5d4-ef209db9b116.png)
Notes section should be placed before the properties section
As we can see in the image above, the notes section is after the properties section, unlike other filters on the same page.
like:
![Screenshot 2022-01-16 030726](https://user-images.githubusercontent.com/80591258/149638502-eb6ab34d-f51b-4326-839c-5f06368ba6fc.png)
&
![Screenshot 2022-01-16 030743](https://user-images.githubusercontent.com/80591258/149638506-e8d5cbf3-6b56-4134-b462-4285cb9de086.png)

files need to be updated in this case:
https://github.com/checkstyle/checkstyle/blob/master/src/xdocs/config_filters.xml
https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithPlainTextCommentFilter.java

**Comment by AbhinavRajputEXE** [comment]
@romani 
please review this issue, if it is valid or not.
there is one more similar kind of issue on the same page and a few typos also.
should I also add them to this same issue?

**Comment by romani** [comment]
yes, please do, we already place Notes above properties in other filters - https://checkstyle.org/config_filters.html#SuppressWithNearbyCommentFilter_Notes

**Comment by AbhinavRajputEXE** [comment]
> the only weird Note is https://checkstyle.org/config_filters.html#SuppressionFilter_Notes that is after Properties section, but it is too big, I am not sure if it make sense to move it above Properties. Lets keep it as is for now.

yes, this was the other one, except these, other filter sections are fine.😅
@romani 
I have one more question
If a filter's module has "Checker" as the parent module, can it suppress checks that have "Treewalker" as their parent module as well? As it is mentioned in the [notes of SuppressWithPlainTextCommentFilter](https://checkstyle.org/config_filters.html#SuppressWithPlainTextCommentFilter_Notes)
```
SuppressionWithPlainTextCommentFilter can suppress Checks that have Treewalker or Checker as parent module.
```
But in its [description](https://checkstyle.org/config_filters.html#SuppressWithPlainTextCommentFilter_Description), it is written that
```
The filter can be used "ONLY" to suppress audit events received from the checks which implement FileSetCheck interface.
In other words, the checks which have Checker as a parent module.
```

**Comment by romani** [comment]
> If a filter's module has "Checker" as the parent module, can it suppress checks that have "Treewalker" as their parent module as well?

yes, but not by Xpath, not very precisely.

>  it is written that

yes, it is mistake, you can fix it also.

**Comment by AbhinavRajputEXE** [comment]
I'm on it

**Comment by romani** [comment]
fix is merged.

@ayushactiveat , thanks a lot for you help
